src/components/index.ts

```diff
@@ -1,27 +1,32 @@
-import PauseButton from './buttons/pause-button';
-import { LevelIndicators } from './level-indicator';
-import PausePopUp from './pause-popup';
-import { PromptText } from './prompt-text';
-import StoneHandler from './stone-handler';
-import { Tutorial } from './tutorial';
-import { Monster } from './monster';
-import { TimerTicking } from './timer-ticking';
-import { Background } from './background';
-import { FeedbackTextEffects } from './feedback-particle-effect/feedback-text-effects';
-import { AudioPlayer } from './audio-player';
-import TrailEffect from './trail-particles';
+import PauseButton from "./buttons/pause-button";
+import { LevelIndicators } from "./level-indicator";
+import PausePopUp from "./pause-popup";
+import { PromptText } from "./prompt-text";
+import StoneHandler from "./stone-handler";
+import { Tutorial } from "./tutorial";
+import { Monster } from "./monster";
+import { TimerTicking } from "./timer-ticking";
+import { Background } from "./background";
+import { FeedbackTextEffects } from "./feedback-particle-effect/feedback-text-effects";
+import { AudioPlayer } from "./audio-player";
+import TrailEffect from "./trail-particles";
+import PlayButton from "./play-button";
+import { AreYouSurePopUp, TextParticle } from "./feedback-particle-effect";
```

Comment:
Remove Surepopup from Feedback Particle effect component as suggested above

Reply:
I move this as well. thanks @ashwinnair-chimple